hazelcast/src/hazelcast/client/compact.cpp

```diff
@@ -1495,6 +1497,7 @@ init_fp_table()
     }
     return FP_TABLE;
 }
+
```

Comment:
is this thread safe?
`static auto FP_TABLE = init_fp_table();`

Reply:
`init_fp_table()` itself is not thread-safe but `static auto FP_TABLE = init_fp_table()` is thread-safe. But I think `static` keyword usage in `init_fp_table()` is unnecessary.